css/portal-dashboard/question-area.less

```diff
@@ -20,9 +20,10 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    flex: 1;
+    flex-grow: 2;
```

Comment:
Why is this needed?  `flex: 1` sets `flex-grow` to 1.  This change will make this box grow at 2 times the rate of other flex boxes in the same container.  I'm wondering if the better approach is to leave this as 1, and then make it so the `rightIcons` class doesn't grow, by using `flex: auto` or setting a width and using `flex-grow: 0`.  I think you might get better results with that.

Reply:
Set the width for the left side title text, and positioned the right side icons.